Module not found: Can't resolve './common.blocksAppRouterAppRouter.js' on windows

Failed to compile.
```
Error in ./src/index.js
Module not found: Can't resolve './common.blocksAppRouterAppRouter.js' in 'c:\work\<PROJECT_NAME>\src'

 @ multi ./~/react-dev-utils/webpackHotDevClient.js ./~/bem-react-scripts/config/polyfills.js ./src/index.js

import AppRouter from 'b:AppRouter';
```

It seems that `b:blah-blah` import missing slash on Windows.

--- a/index.js
+++ b/index.js
@@ -35,6 +35,7 @@ module.exports = function(source) {
         }, {}),
         defaultExts = Object.keys(extToTech),
         allPromises = [],
+        winPath = path => path.replace(/\\/g, '/'),
         namingOptions = options.naming || 'react',
         bemNaming = bn(namingOptions);
 
@@ -82,7 +83,7 @@ module.exports = function(source) {
                         cell : bemCell,
                         exist,
                         // prepare path for require cause relative returns us string that we couldn't require
-                        path : requiredPath(path.relative(path.dirname(this.resourcePath), entityPath))
+                        path : winPath(requiredPath(path.relative(path.dirname(this.resourcePath), entityPath)))
                     };
                 });
         });

